spec/spec_helper.rb

```diff
@@ -83,6 +80,11 @@
   end
 
   config.before(:each) do
+    # mock HT member data to use in tests. Tests may change this data, so 
```

Comment:
I don't know why this worked before. What I observed was that data added with `add_temp` leaked between tests and caused issues. This seemed like a reasonable fix.